docroot/themes/humsci/humsci_basic/src/scss/components/_masthead.scss

```diff
@@ -25,6 +25,49 @@
         margin-bottom: 0;
       }
     }
+
+    &__line1,
+    &__line2 {
```

Comment:
Do we need the font-style normal that they were adding to `.su-lockup__line-2`?

Reply:
When I inspect line2, I see that font-style: normal is already there and inheritied. I'm not sure why they even have it in the asset injector settings.  However if we want to add it to cover out bases we could.